guppylang/prelude/builtins.py

```diff
@@ -21,19 +21,28 @@
     ReversingChecker,
     UnsupportedChecker,
     float_op,
-    hugr_float_type,
-    hugr_int_type,
     int_op,
     logic_op,
 )
-from guppylang.tys.builtin import bool_type_def, linst_type_def, list_type_def
+from guppylang.tys.builtin import (
+    bool_type_def,
+    float_type_def,
+    int_type_def,
+    linst_type_def,
+    list_type_def,
+    nat_type_def,
+)
 
 builtins = GuppyModule("builtins", import_builtins=False)
 
 T = guppy.type_var(builtins, "T")
 L = guppy.type_var(builtins, "L", linear=True)
 
 
+# Define the nat type so scripts can import it
+nat = nat_type_def
```

Comment:
I'd expect this to implement some numeric dunder methods, like `__add__`, `__eq__`, `__gt__`?